sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql

```diff
@@ -344,6 +344,114 @@ metrics_org_mozilla_ios_fennec AS (
   WHERE
     mozfun.norm.truncate_version(client_info.app_display_version, 'major') >= 28
 ),
+-- metrics for Focus Android Baseline release
+metrics_org_mozilla_focus AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    client_info.client_id,
+    normalized_country_code AS country,
+    'Focus Android Baseline' AS app_name,
```

Comment:
as per the sql-generators[readme](https://github.com/mozilla/bigquery-etl/tree/main/sql_generators#readme), we do not require generating to merge the query.sql to main as it will be generated via CI

Reply:
@alekhyamoz good catch - I think this can be done in a follow-up PR along with any other generated queries